### DISCUSSION TITLE
Chrome beta version

### DISCUSSION THREAD

**Discussion by alexardra** [vuejs/devtools-v6#2083] [Q&A]
Looks like the link to beta version of devtools for chrome is broken here: https://devtools.vuejs.org/guide/installation.html#chrome

I am not able to find it in chrome extension search either. Can anyone help with that?

**Comment by spacemelon1** [comment]
I tried accessing it and comes up with a 404 error, could likely have been removed by google

**Comment by RinkAttendant6** [comment]
It has been flagged as malware by Microsoft Edge (and probably Google):

![Vue.js devtools malware](https://github.com/vuejs/devtools/assets/351158/4c33b3ec-ae9a-495b-b1f8-b32a5b9d22f3)

**Comment by atymic** [comment]
https://github.com/vuejs/devtools/issues/2099
Created an issue
